test/test-fs.c

```diff
@@ -832,7 +837,7 @@ static void utime_cb(uv_fs_t* req) {
   ASSERT(req->fs_type == UV_FS_UTIME);
 
   c = req->data;
-  check_utime(c->path, c->atime, c->mtime);
+  check_utime(c->path, c->atime, c->mtime, 0);
```

Comment:
```suggestion
  check_utime(c->path, c->atime, c->mtime, /* test_lutime */ 0);
```
And ditto elsewhere. :-)